test_communication/test/test_publisher_subscriber_python.py.in

```diff
@@ -0,0 +1,38 @@
+import sys
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler, default_exit_handler
+from launch.launcher import DefaultLauncher
+
+
+def test_publisher_subscriber():
+    ld = LaunchDescriptor()
+
+    ld.add_process(
+        cmd=[sys.executable,
+             '@TEST_PUBLISHER_PY_EXECUTABLE@',
+             '@TEST_MESSAGE_TYPE@',
+             '-r', '@TEST_RMW_IMPLEMENTATION_PY@'],
```

Comment:
#126 has an implementation already setting an implementation specific env variable. That approach would dovetail well here.